### PR TITLE
Move from pytest-cov to coverage

### PR DESCRIPTION
**Pull request by elParaguayo**
pytest-cov 4.0.0 dropped support for `multiprocessing` which massively impacts our coverage scores. Pinning to 3.0.0 would retain that support but is not a good long term solution.

pytest-cov recommends moving to coverage but I'm currently seeing a few drops in coverage on a small number of tests. Needs debugging.

### CODE REVIEW

**Comment by elParaguayo** [comment]
Asked for some support in debugging the differences here: https://github.com/nedbat/coveragepy/issues/1458

File: tox.ini [hunk 1]
```diff
@@ -48,9 +48,16 @@ commands =
     {toxinidir}/scripts/ffibuild
 # py310 currently fails with -W error, see: https://gitlab.gnome.org/GNOME/pygobject/-/issues/476
```
Comment (m-col) [inline tox.ini:49]: This gnome issue has been resolved now so we no longer need a separate `py310` run. Might be worth updating that while we're here?

**Review by m-col — APPROVED** [review]

File: tox.ini (hunk 1 above)
Comment (elParaguayo) [reply tox.ini]: Still got an error.

**Comment by elParaguayo** [comment]
Coverage actually looks ok.

**Comment by elParaguayo** [comment]
Most of the tests that have reduced coverage are ones that inherit `base.ThreadPoolText`. It looks like the `poll` method is no longer being covered in the `init_configure` test, perhaps because this is called by `qtile.run_in_executor`.

**Pull request merged**